data/pb/BillingService/TaxPeriod.json

```diff
@@ -169,6 +169,94 @@
       "service": "EXPENSE.INCENTIVES",
       "code": "EXPENSEINCENTIVES",
       "financialYear": "2022-23"
+    },
+    {
+      "fromDate": 1680244199000,
+      "toDate": 1711866599000,
+      "periodCycle": "ANNUAL",
+      "service": "EXPENSE.ELECTRICITY_BILL",
+      "code": "EXPENSEELECTRICITY_BILL",
+      "financialYear": "2023-24"
+    },
+    {
+      "fromDate": 1680244199000,
+      "toDate": 1711866599000,
+      "periodCycle": "ANNUAL",
+      "service": "EXPENSE.SALARY",
+      "code": "EXPENSESALARY",
+      "financialYear": "2023-24"
+    },
+    {
+      "fromDate": 1680244199000,
+      "toDate": 1711866599000,
+      "periodCycle": "ANNUAL",
+      "service": "EXPENSE.MACHINERY_REPAIR",
+      "code": "EXPENSEMACHINERYREPAIR",
+      "financialYear": "2023-24"
+    },
+    {
+      "fromDate": 1680244199000,
+      "toDate": 1711866599000,
+      "periodCycle": "ANNUAL",
+      "service": "EXPENSE.OTHERS",
+      "code": "EXPENSEOTHERS",
+      "financialYear": "2023-24"
+    },
+    {
+      "fromDate": 1680244199000,
+      "toDate": 1711866599000,
+      "periodCycle": "TWOYEARS",
```

Comment:
@rahuldevgarg-egov @Saloni-eGov WHat is the use of periodCycle in MDMS. Please check this.

Reply:
```
if (taxPeriodCriteria.getPeriodCycle() != null) {
			if (filterExpression.length() != 0)
				filterExpression.append(" && ");
			filterExpression
					.append(TAXPERIOD_PERIODCYCLE_FILTER.replace("VAL", taxPeriodCriteria.getPeriodCycle().toString()));
		}
```
it is used in MDMS service for getting List of TaxPeriod